<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_dialog</class>
 <widget class="QDialog" name="Main_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>6</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Main_btn_CreateNum">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>480</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>로또 번호 생성</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Main_btn_AnalysisNum">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>480</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>로또 번호 분석</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lotto_round_display">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>60</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>문체부 제목 바탕체</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>회차</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textedit_html_viewer">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="test_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>추천 번호</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
